packages/editor/src/store/actions.js

```diff
@@ -192,6 +195,19 @@ export const editPost =
 			.editEntityRecord( 'postType', type, id, edits, options );
 	};
 
+/**
+ * Action that opens an editor sidebar.
+ *
+ * @param {?string} name Sidebar name to be opened.
+ */
+export const openGeneralSidebar =
```

Comment:
This action will become public, we should probably avoid that? Why do we need this shortcut?